packages/common/src/directives/ng_optimized_image/ng_optimized_image.ts

```diff
@@ -223,6 +224,10 @@ export class NgOptimizedImage implements OnInit, OnChanges, OnDestroy {
       assertRequiredNumberInput(this, this.width, 'width');
       assertRequiredNumberInput(this, this.height, 'height');
       assertValidLoadingInput(this);
+      const loadListener = this.renderer.listen(this.imgElement.nativeElement, 'load', () => {
+        loadListener();
```

Comment:
I'd propose changing the name of this const to better reflect what it contains as a value.
```suggestion
      const removeListenerFn = this.renderer.listen(this.imgElement.nativeElement, 'load', () => {
        removeListenerFn();
```